compiler/nimblecmd.nim

```diff
@@ -134,3 +156,75 @@ proc nimblePath*(conf: ConfigRef; path: AbsoluteDir, info: TLineInfo) =
   if i != -1:
     conf.nimblePaths.delete(i)
   conf.nimblePaths.insert(path, 0)
+
+when isMainModule:
```

Comment:
move tests to tests/compiler/tnimblecmd.nim

Reply:
@timotheecour Done.

Reply:
I still see them here